Gemfile

```diff
@@ -26,7 +26,7 @@ gem "aasm", "~> 4.11.1"
 gem "paperclip", "~> 4.2.0"
 gem "paper_trail"
 gem "awesome_nested_set", "~> 3.1.3"
-gem "nokogiri", "~> 1.7.1"
+gem "nokogiri"
```

Comment:
Hi Robin--in regards to your email about specifying the gem version in the Gemfile, one of the main reasons we do it this way in NUCore is so that when we update a gem in open, the update also gets to the dependent forks (nucore-nu, nucore-dartmouth, nucore-uic). Since the various forks don't share a Gemfile.lock, pinning it in the Gemfile has been a good way to manage it.

If you don't like `~>`, maybe `>=` would be better?